src/screens/GiftCard/GiftCardDetail.tsx

```
@@ -0,0 +1,118 @@
+        <Text style={styles.errorText}>올바르지 않은 금액권 ID입니다.</Text>
+      </View>
+    );
+  }
+
+  const gift = giftCardData[giftId];
+  const router = useRouter();
+
+  if (!gift) {
+    return (
+      <View style={styles.container}>
+        <Text style={styles.errorText}>금액권 정보를 찾을 수 없습니다.</Text>
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)